src/OpenLoco/Audio/Audio.cpp

```diff
@@ -1052,25 +1057,7 @@ namespace OpenLoco::Audio
 
         if (playlist.size() == 0)
         {
-            if (excludeTrack == kNoSong)
-            {
-                for (auto i = 0; i < kNumMusicTracks; i++)
-                {
-                    if (i != excludeTrack)
-                    {
-                        playlist.push_back(i);
-                    }
-                }
-            }
-            else
-            {
-                const auto& mi = kMusicInfo[excludeTrack];
```

Comment:
I removed this 'else' as it doesn't really do anything useful. better off to just try to get the songs from the enum, and if that fails, play all songs

Reply:
Where do the start and end years get taken into account now, then?

Reply:
in the `case case MusicPlaylistType::currentEra:` on line 1028 or so

Reply:
The way i read it is that this else branch was for custom playlist users. If there are no custom tracks selected it would fall back to the currentEra playlist. Perhaps we should make it like the following:

```
switch (playlist){
 case ...
}

if (playlist.empty()){
  switch(playlist){
    case custom:
       callThisFunction(currentEraPlaylist);
       break;
    case currentEra:
       callThisFunction(allPlaylist);
       break;
    default:
       break;
}
```
i.e. if the playlist has no songs degrade to the next playlist custom -> currentEra -> allSongs -> giveUp.

Reply:
Fair enough, I've re-added the first fallback case to currentEra, and then it falls back to allSongs if nothing else. If there truly are no songs this function will still crash, but that would also mean there are bigger problems (music objs have been deleted AND the game loaded successfully without them)